fe/fe-core/src/main/java/com/starrocks/lake/StarOSAgent.java

```diff
@@ -448,8 +449,23 @@ public Set<Long> getBackendIdsByShard(long shardId) throws UserException {
                             .getBackendIdWithStarletPort(pair[0], Integer.parseInt(pair[1]));
 
                     if (backendId == -1L) {
-                        LOG.warn("backendId for {} is -1", workerAddr);
-                        continue;
+                        LOG.info("can't find backendId with starletPort for {}.", workerAddr);
```

Comment:
print starletPort?

Reply:
starletport is parsed from workerAddr which is already logged.

Reply:
bePort is printed

Reply:
i know, beport is not implied in `workerAddr`, so need to be printed in log, starletPort is already implied in workerAddr, no need to print again. 

`can't find backendId with starletPort` means it can't find the BE node with given host and starletPort, not necessary mean "here is the starletPort I used to search`.